### DISCUSSION TITLE
Is it possible to whitelist wallets in wallets select modal?

### DISCUSSION THREAD

**Discussion by start940315** [WalletConnect/walletconnect-monorepo#1945] [Developer Support]
There are too much wallets the default wallets.json api returns. I need to choose some of them to show to the users. Is there any way to do it?

**Comment by chadyj** [comment]
Does this help? https://docs.walletconnect.com/2.0/web3modal/configuration#mobilewallets-optional

**Reply by start940315** [reply]
no it's not working
